package.json

```diff
@@ -534,6 +537,8 @@
     "./RichTextEditor": "./RichTextEditor.js",
     "./Scroller": "./Scroller.js",
     "./Select": "./Select.js",
+    "./SideNav": "./SideNav.js",
+    "./SideNavItem": "./SideNavItem.js",
```

Comment:
Was it intentionally included in this PR? Doesn't look like a related change.

Reply:
🤔

Reply:
I guess, it was done by a script, because we already have `SideNav` on line 421, but no supporting export on 538. I would suggest to leave it here because we would need to include it here anyway (and probably it's gonna be generated on each script run).